rainforest/environments.go

```diff
@@ -2,22 +2,25 @@ package rainforest
 
 // EnvironmentParams are the parameters used to create a new Environment
 type EnvironmentParams struct {
-	Name string `json:"name"`
-	URL  string `json:"url"`
+	Name        string `json:"name"`
+	URL         string `json:"url"`
+	IsTemporary bool   `json:"is_temporary"`
 }
 
 // Environment represents an environment in Rainforest
 type Environment struct {
-	ID   int    `json:"id"`
-	Name string `json:"name"`
+	ID          int    `json:"id"`
+	Name        string `json:"name"`
+	IsTemporary bool   `json:"is_temporary"`
```

Comment:
Do we use the pattern of `is_XYZ` in the API already, or do we tend to just use `XYZ`?

Reply:
We do:

<img width="813" alt="image" src="https://user-images.githubusercontent.com/44345/131334957-13d2a8a5-4996-4979-a32d-2b0c1dd6b49c.png">